cmd/dep/init.go

```diff
@@ -152,7 +163,20 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	l = dep.LockFromSolution(soln)
 
 	rootAnalyzer.FinalizeRootManifestAndLock(m, l)
-	gs.FinalizeRootManifestAndLock(m, l)
+	if cmd.gopath {
+		gs.FinalizeRootManifestAndLock(m, l)
+	} else {
+		// Pick the direct dependencies from the above solution and add to manifest
```

Comment:
To be honest I'm not entirely sure what's the right approach. 😊 I was just asking because I was confused.

What I am pretty sure about is:

* The logic should be in `rootAnalyzer.FinalizeManifestAndLock` instead of the init command directly.
* When someone runs `dep init -skip-tools -gopath` it shouldn't result in a populated lock and an empty manifest.